py3status/modules/xrandr.py

```diff
@@ -399,11 +402,15 @@ def xrandr(self):
         self._set_available_combinations()
         self._choose_what_to_display()
 
-        if self.fixed_width is True:
-            output = self._center(self.displayed)
+        if (len(self.available_combinations) < 2 and
+                self.hide_if_single_combination):
+            full_text = ''
```

Comment:
See `./py3status/tests/test_formatter.py` for various examples.

We make adjustment in `xrandr` code to pass out number of outputs as `{count}` or `{output_num}` placeholder. None of that hide options. Simple raw code.

If you don't want single combination, we use the formatter to tweak things away to our desires... I imagine it would be something like this.
`format = [\?more_than_one=count {output}]` _Maybe bad name. not actual example_.

We already can do this kind of this thing now if we keep getting more formatter options in the future, it's totally doable. Also, this is nice because it would just work with all modules instead.

I'm assuming we want to deprecate `fixed_width` in this code eventually once we get a working formatter for `fixed_width` so we just use `[\?fixed_width {output}]` instead.

IDEAL GOAL: 
```
count = len(self.available_combinations)
self.py3.safe_format(self.format, {'output': output, 'count': count})
```
```
format = [\?more_than_one=count {output}]
```
The `[{output}]` thing is already in the formatter to hide values if we don't get any. No need to worry about that. Really, this is confusing for me too because I don't know what quite works well. There are more wish features in here though. https://github.com/ultrabug/py3status/issues/752 😄